requirements.txt

```diff
@@ -1,8 +1,5 @@
-oldest-supported-numpy
```

Comment:
I guess I don't need this magic package for a python-only thing, but we should declare numpy, right?

Reply:
yep! fixed!